src/main/java/ch/wisv/events/core/model/order/Customer.java

```diff
@@ -25,51 +23,40 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+@Data
 @Entity
-@EqualsAndHashCode
 public class Customer {
 
     /**
      * Field id of the customer.
      */
     @Id
     @GeneratedValue
-    @Getter
     private Integer id;
```

Comment:
You can use @Setter(AccesLevel.NONE) to prevent a setter being generated by Lombok